src/components/common/form/OtpInput.js

```diff
@@ -52,26 +54,15 @@ const getSingleOtpInputStylesFromProps = ({ theme }) => ({
     display: 'flex',
     alignItems: 'center',
     flex: 1,
-    paddingHorizontal: Platform.select({
-      // FIXME: RN
-      default: 0,
-      web: '0.4rem',
-    }),
+    paddingHorizontal: getDesignRelativeWidth(4),
```

Comment:
you need to add second param false if you want padding to grow for larger>design screens